dotnet/src/webdriver/Chrome/ChromeDriverService.cs

```diff
@@ -46,14 +47,15 @@ private ChromeDriverService(string executablePath, string executableFileName, in
         /// <returns>A ChromeDriverService that implements default settings.</returns>
         public static ChromeDriverService CreateDefaultService()
         {
-            return CreateDefaultService(new ChromeOptions());
+            return new ChromeDriverService(null, null, PortUtilities.FindFreePort());
```

Comment:
I don't think `null` is allowed as a service path. I quickly looked through further usages of it and, seems, `null` is not valid argument here: https://github.com/SeleniumHQ/selenium/blob/4eeed646fdb3ef31ad3f712f7ebd852b2ca849ee/dotnet/src/webdriver/DriverService.cs#L267

Reply:
The idea of this is that if the user passed something in to use it. Otherwise we need to set it from the options. We only know both in driver constructor, so we always make sure it isn't null before it gets used.